pages/components/cards.js

```diff
@@ -126,18 +147,15 @@ export default function Cards({ posts, setPosts }) {
                 />
               </DropdownButton>
               <DropdownContent>
-                <DropdownButton edit onClick={() => handleContentEdit(post.id)}>
+                <DropdownButton onClick={() => handleEditComment(post.id)}>
```

Comment:
try to be explicit when working with buttons about the type of the button:

```suggestion
                <DropdownButton type="button" onClick={() => handleEditComment(post.id)}>
```